libs/wingsdk/src/util/util.ts

```diff
@@ -223,11 +222,11 @@ export class Util {
    * @param opts `ExecOptions`, such as the working directory and environment variables.
    * @returns A struct containing `stdout`, `stderr` and exit `status` of the executed program.
    */
-  public static async exec(
+  public static exec(
```

Comment:
@Chriscbr what do you think? It's not ideal that exec will be sync in inflight.

Is there a plan to support preflight/inflight variants of utils?

Reply:
I agree. Right now the rough plan is to allow unphased functions to have separate preflight and inflight implementations, at least if they're implemented with JavaScript. Check out the RFC we have open for more details: https://github.com/winglang/wing/pull/1711

For now I'm cool with trying out this change just to be pragmatic since I'm not we support this with our TypeScript-based SDK (but it will be nice to get this working eventually)